MANIFEST.in

```diff
@@ -1,6 +1,13 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
+include *.rst
```

Comment:
Do we need this directive when later on we  do a recursive include?

Also not sure whether this should be raised here or in the meta repo.

Reply:
We need this because this includes `*.rst` at the root. The later recursive include applies to the `src` subfolder.

Reply:
Thank you!